login-module/test/browser/run.js

```diff
@@ -0,0 +1,61 @@
+/* eslint-env node, es6 */
```

Comment:
Boilerplate code to start Puppeteer, open the `index.html` page and intercepts logs and fetch tests results from mocha.